### ISSUE TITLE
newer version of closure-linter-wrapper

### ISSUE THREAD

**Issue by yisding** [TomSeldon/gulp-gjslint#15]
Could we get the v1.0.1 of closure-linter-wrapper? I'm not sure whether there are any API changes, but it has the latest version of gjslint.

**Comment by TomSeldon** [comment]
Hi @yisding,

I'll take a look at this today and see if it causes any issues. If it all looks ok, I'll update the dependency and get a new release out.

From their release notes, it seems the only (documented) breaking change is that they've dropped support for the `ignore_errors` linting flag.

**Comment by TomSeldon** [comment]
I tried bumping the version to 1.0.1 and all of the tests still pass. I also tried running the linter against a very large code base where I'm currently using the older version of the linter and everything works as expected (plus it's picked up on some linting mistakes that the previous version completely missed).

I've merged in the change, bumped the version **v0.1.5** and pushed a release to NPM. If you notice any weirdness then let me know.

Thanks :)
